snapcraft/plugins/v2/python.py

```diff
@@ -129,6 +129,9 @@ def get_build_commands(self) -> List[str]:
             'SNAPCRAFT_PYTHON_PATH="$(readlink -e "${SNAPCRAFT_PYTHON_PATH}")"',
             '"${SNAPCRAFT_PYTHON_PATH}" -m venv ${SNAPCRAFT_PYTHON_VENV_ARGS} "${SNAPCRAFT_PART_INSTALL}"',
             'SNAPCRAFT_PYTHON_VENV_INTERP_PATH="${SNAPCRAFT_PART_INSTALL}/bin/${SNAPCRAFT_PYTHON_INTERPRETER}"',
+            'SNAPCRAFT_PYTHON_BUILD_EXTRAS="$(readlink -f "../python-build-extras")"',
```

Comment:
This should be conditional to not having wheel staged.
How would I lock down a wheel?

Might be worth using some python logic to check python-packages and conditionally extend the list

Reply:
As noted in the LP ticket, it is desirable to not install wheel (as a build time dependency, not run time).

A couple of thoughts, we can (a) honor constraints, (b) introduce `python-wheel-version` property, (c) introduce `python-build-packages` property which gets default populated with `wheel`, (d) add wheel to `python-packages` and the publisher can filter it out if desired.